File: 2.0/java/contacts/updateDncNumber.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.contacts.model.DoNotContact;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        DoNotContact dnc = new DoNotContact();
        dnc.setCall(true);
        dnc.setListId(1975140003L);
        dnc.setNumber("12563505052");
        dnc.setText(true);
        client.dncApi().update(dnc);
    }
}

File: 2.0/csharp/contacts/updateDncNumber.cs
using CallfireApiClient;
using CallfireApiClient.Api.Contacts.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var dnc = new DoNotContact
        {
            ListId = 1975140003,
            Text = true,
            Call = true,
            Number = "12563505052"
        };
        client.DncApi.Update(dnc);
    }
}
